tests/integration/release/data/parameters.csv

```diff
@@ -0,0 +1,7 @@
+dataset,engine,memory,time
+acs,pandas,1,00:00:01
```

Comment:
So this is both test params (dataset, enging) and slurm requiests (memory, time)?

Why only 1gb and 1 min?

And what about the OTHER test params (weren't there like 7 or 8 of them)?

Reply:
These are just placeholders for now - actual values will be in next PR